glmtools/glm.py

```diff
@@ -1,10 +1,11 @@
 """ Functions for running GLM on 2D and 3D data
 """
-
 import numpy as np
 import numpy.linalg as npl
-
-
+import matplotlib.pyplot as plt
+# Print array values to 4 decimal places
+np.set_printoptions(precision=4)
```

Comment:
You don't need this line - that is only for printing interactively - useful for the tutorials to avoid clutter, but confusing in code, because it changes the printoptions for any module importing numpy, which might be surprising.